From 5cdc97ce02770516db75c03a2ba5aac248b1e4b8 Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Fri, 21 May 2021 23:03:54 +0200
Subject: [PATCH 17/N] Use msys tools

Some of the external tools pacman uses might unfortunately have incompatible
versions in PATH, e.g. when using makepkg-mingw.  Let's always invoke them using
absolute paths.  Most of the patching is currently done in
0000-pacman-msysize.patch, these are just some new occurrences.  Ideally these
PATH guards will be a completely separate patch in the future.
---
 scripts/libmakepkg/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scripts/libmakepkg/meson.build b/scripts/libmakepkg/meson.build
index b0697f7..3bdc66c 100644
--- a/scripts/libmakepkg/meson.build
+++ b/scripts/libmakepkg/meson.build
@@ -7,6 +7,7 @@ libmakepkg_modules = [
   { 'name' : 'lint_pkgbuild', 'has_subdir' : true },
   { 'name' : 'source',        'has_subdir' : true },
   { 'name' : 'srcinfo',                           },
+  { 'name' : 'utils_fixed_path',                  },
   { 'name' : 'tidy',          'has_subdir' : true },
   { 'name' : 'util',          'has_subdir' : true },
 ]
-- 
2.31.1

